bin/src/run/watch.ts

```diff
@@ -148,7 +148,7 @@ export default function watch(
 					break;
 
 				case 'END':
-					if (!silent && isTTY) {
+					if (!silent) {
```

Comment:
It seems this check was initially added exactly for TTY support - https://github.com/rollup/rollup/pull/1574.

Can you perhaps explain why you think this should be the other way around?

Reply:
Sorry, I mean this line was initially added exactly for Lerna support :)

Reply:
Exact link - https://github.com/rollup/rollup/pull/1574/files#diff-4a84b75586b4af29e839a9837aa7838eR79